recording-oracle/src/utils/manifest.ts

```diff
@@ -0,0 +1,29 @@
+import Joi from 'joi';
+
+import { SUPPORTED_EXCHANGE_NAMES } from '@/common/constants';
+import { type CampaignManifest } from '@/modules/campaigns/types';
```

Comment:
```suggestion
import type { CampaignManifest } from '@/modules/campaigns';
```
But we will also need to export this type from there